index.js

```diff
@@ -156,22 +183,34 @@ function getOptionsArgument(serviceOrOptions?: string | Options) {
     : serviceOrOptions;
 }
 
+function getMinimumSecurityLevel(minimumSecurityLevel?: SecMinimumLevel) {
+    if (minimumSecurityLevel === undefined) {
+        return SECURITY_LEVEL.ANY;
+    } else {
+        return minimumSecurityLevel
+    }
+}
+
 /**
  * Saves the `username` and `password` combination for `service`.
  * @param {string} username Associated username or e-mail to be saved.
  * @param {string} password Associated password to be saved.
+ * @param {string} minimumSecurityLevel `SECURITY_LEVEL` defines which security
+ *                 level is minimally acceptable for this password.
  * @param {string|object} serviceOrOptions Reverse domain name qualifier for the service, defaults to `bundleId` or an options object.
  * @return {Promise} Resolves to `true` when successful
  */
 export function setGenericPassword(
   username: string,
   password: string,
+  minimumSecurityLevel?: SecMinimumLevel,
```

Comment:
this is a breaking change (https://github.com/oblador/react-native-keychain/pull/176/files#r243020977); can you move this to Options?

Reply:
done